common/src/main/java/com/wynntils/features/utilities/PerCharacterGuildContributionFeature.java

```diff
@@ -73,6 +78,30 @@ public void onChatReceived(ChatMessageReceivedEvent event) {
 
             characterContributions.get().put(Models.Character.getId(), contributionAmount);
             characterContributions.touched();
+
+            sendContributionMessage(contributionAmount);
         }
     }
+
+    private void sendContributionMessage(int contributionAmount) {
+        MutableComponent contributionMessage = Component.literal(ChatFormatting.DARK_AQUA + "You will now contribute "
+                + ChatFormatting.AQUA
+                + contributionAmount + "%" + ChatFormatting.DARK_AQUA
+                + " of your XP to " + ChatFormatting.AQUA + Models.Guild.getGuildName() + ChatFormatting.DARK_AQUA
+                + ".\n");
+
+        contributionMessage.append(Component.literal("This will only apply to your current character.\n")
+                .withStyle(ChatFormatting.DARK_AQUA));
+
+        contributionMessage.append(Component.literal("Click here")
+                .withStyle(ChatFormatting.AQUA)
+                .withStyle(ChatFormatting.UNDERLINE)
+                .withStyle(style -> style.withClickEvent(new ClickEvent(
+                        ClickEvent.Action.RUN_COMMAND, "/feature disable PerCharacterGuildContributionFeature"))));
```

Comment:
I thought it was just `/feature disable PerCharacterGuildContribution`?

Reply:
Also use /wynntils feature

Reply:
Pretty sure it didn't work without Feature at the end but will double check

Reply:
Yeah needs Feature on the end
![2023-08-27_11 40 09](https://github.com/Wynntils/Artemis/assets/8337467/ceb378c3-d982-4bda-9c43-b41501c22b11)

Reply:
Oh. That sounds like a bug to me..! But okay, let's do it like this for now and hope we'll remember to update this instance when fixing the bug.